contrib/utilities/makeofflinedoc.sh

```diff
@@ -24,19 +24,16 @@ then
   exit 1
 fi
 
-mkdir -p images
+echo "Patching html files ..."
+sed -i 's#"https\?://www.dealii.org/images/#"images/#g' step_*.html class*.html
```

Comment:
I'm coming late to the game, but is the backslash in front of the question mark a mistake? Or is this really how sed wants this?